src/applications/disability-benefits/4142/app-entry.jsx

```diff
@@ -0,0 +1,14 @@
+import '../../../platform/polyfills';
```

Comment:
@imeecuison : Why is this file named "app-entry.jsx" vs "form-entry.jsx" or "4142-entry.jsx" . I would recommend that we don't come up with a new name pattern for the entry file.

Reply:
The entry name is inferred based on the questions you answer in the form generator. The pattern provided is the `-entry` the put on the filename for you.

Reply:
This name was generated with the new version of the generator. I'm reluctant to change it at this time due to not knowing what this will affect.